### PR TITLE
Attendance display

### PR DESCRIPTION
**Pull request by Ellieyee**
-Add attendance feature into GUI
-currently is can only be displayed through the markattendancecommand
-In order to reduce unintended regressions, the displayAttendanceCommand will be created in the next PR.
- Changes are made to commandResult to pass desired attendance to display (will be further improved in the future)

### CODE REVIEW

File: src/main/java/seedu/tarence/logic/LogicManager.java [hunk 1]
```diff
@@ -71,7 +71,7 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
             combinedFeedback.append(currCommandResult.getFeedbackToUser() + "\n");
 
             // check for exit/help condition
-            if (currCommandResult.isExit() || currCommandResult.isShowHelp()) {
+            if (currCommandResult.isExit() || currCommandResult.isShowHelp() || currCommandResult.isShowAttendance()) {
```
Comment (ZiHawkEye) [inline src/main/java/seedu/tarence/logic/LogicManager.java]: why is this necessary?

File: src/main/java/seedu/tarence/logic/commands/CommandResult.java [hunk 2]
```diff
@@ -17,13 +19,20 @@
     /** The application should exit. */
     private final boolean exit;
 
+    /** The application should display attendance */
+    private boolean attendancedisplay;
```
Comment (ZiHawkEye) [inline src/main/java/seedu/tarence/logic/commands/CommandResult.java]: Should adhere to naming conventions I think eg. hasAttendanceDisplay

File: src/main/java/seedu/tarence/logic/commands/MarkAttendanceCommand.java [hunk 3]
```diff
@@ -123,8 +123,10 @@ public CommandResult execute(Model model) throws CommandException {
         model.setAttendance(targetTutorial, week, targetStudent);
 
         String isPresent = targetTutorial.getAttendance().isPresent(week, targetStudent) ? "present" : "absent";
-        return new CommandResult(String.format(MESSAGE_MARK_ATTENDANCE_SUCCESS,
+        CommandResult commandResult = new CommandResult(String.format(MESSAGE_MARK_ATTENDANCE_SUCCESS,
                 targetStudent.getName(), isPresent));
+        commandResult.setAttendanceToDisplay(targetTutorial);
```
Comment (ZiHawkEye) [inline src/main/java/seedu/tarence/logic/commands/MarkAttendanceCommand.java]: Perhaps it might be better to have another constructor for CommandResult - CommandResult(String, Tutorial) so that immutability could be maintained without setAttendanceToDisplay()?

Comment (Ellieyee) [reply src/main/java/seedu/tarence/logic/commands/MarkAttendanceCommand.java]: oo thanks for the feedback (the ones above too) 
I'll update the PR and notify when it's ready
